client/js/app/utils/ExplorerUtils.js

```diff
@@ -79,36 +82,34 @@ module.exports = {
     }
     var params = _.cloneDeep(explorer.query);
 
-    // Set the timeframe (will get removed if it's null o undefined)
-    params.timeframe = module.exports.getTimeframe(explorer);
```

Comment:
@aroc if you pass in a null or undefined time value, you will get a null timeframe back... which then means it gets plucked by `FormatUtils.isValidQueryValue` later on.